.circleci/config.yml

```diff
@@ -33,7 +33,7 @@ jobs:
       - run:
           name: make tests
           command: make tests
-          no_output_timeout: 30m
+          no_output_timeout: 45m
```

Comment:
This branch does not have the optimized test coverage yet. I will revert after merging.